examples/js/controls/TrackballControls.js

```diff
@@ -96,8 +96,12 @@ THREE.TrackballControls = function ( object, domElement ) {
 			this.screen.top = box.top + window.pageYOffset - d.clientTop;
 			this.screen.width = box.width;
 			this.screen.height = box.height;
+			
 
 		}
+		if(!this.screen.width && !this.screen.height) {
```

Comment:
Sorry I missed this message. I have lost the context but maybe the reason behind using `!this.screen.width` was to catch both `0` and `undefined` in case in some circumstances browsers return `undefined`. Anyways it is very minor but as I remember it took me a long time to figure out why trackball was breaking.